src/nostd.h

```diff
@@ -1,25 +1,11 @@
 #ifndef __NES_DIFF_NO_STD
 #define __NES_DIFF_NO_STD
 
-int strcmp(const char* buffer1, const char* buffer2);
+int strcmp(const char *l, const char *r)
 
-int strcmp(const char* buffer1, const char* buffer2)
+int strcmp(const char *l, const char *r)
 {
-    register int compare_buffers;
-
-    while (*buffer1 && *buffer2)
-    {
-        compare_buffers = (*buffer1) - (*buffer2);
-
-        if (compare_buffers != 0) {
-            return compare_buffers;
-        }
-
-        if (*buffer1) buffer1++;
-        if (*buffer2) buffer2++;
-    }
-
-    return 0;
+	for (; *l==*r && *l; l++, r++);
```

Comment:
could change the `;` at the end of the loop to `{}`, I like to use it like that, but unfortunately I found that some compilers don't support it.